### PR TITLE
Gherkin Scripts Helm Release Scenarios - [ODC-2993]

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fixes:** https://issues.redhat.com/browse/ODC-4075

**Problem:** Gherkin Scripts Design for Helm Release epic ODC-2993

**Solution:** These are the gherkin scripts designed covering all the scenarios related to Helm Release epic ODC-2993

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Feature: Helm Chart
+    User will be able to updated the chart versions of the helm charts
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature:2]:
```suggestion
    User will be able to update the chart versions or values config of a helm release
```

Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/Helm/helm-Chart-Version-ODC-3022.feature:2]: *update

**Pull request merged**